recipe/build_base.sh

```diff
@@ -97,6 +97,12 @@ if [[ ${_OPTIMIZED} = yes ]]; then
   CXXFLAGS=$(echo "${CXXFLAGS}" | sed "s/-O2/-O3/g")
 fi
 
+if [[ ${PY_INTERP_DEBUG} == yes ]]; then
+  CPPFLAGS=$(echo "${CPPFLAGS}" | sed "s/-O2/-O0/g")
+  CFLAGS=$(echo "${CFLAGS}" | sed "s/-O2/-O0/g")
+  CXXFLAGS=$(echo "${CXXFLAGS}" | sed "s/-O2/-O0/g")
+fi
+
```

Comment:
Does it make sense to disable compile time optimizations for debug builds?

Reply:
Yes. There are cases in which an optimized debug build does make sense, but an `-O0` build is probably what most would expect.